tegel/src/components/theme/colour/colour-brand.stories.js

```diff
@@ -0,0 +1,32 @@
+export default {
+  title: 'Foundations/Colour',
+  parameters: {
+    layout: 'fullscreen',
+  },
+};
+
+const Template = () => `
+  <style>
+  .demo-wrapper {
+    height: 90px;
+  }
+  .demo-wrapper span {
+    color: white;
+    background-color: black;
+    border: 1px solid white;
+    padding: 4px;
+    position: absolute;
+  }
+</style>
+  <div class="demo-wrapper" style="background-color: var(--sdds-black)">
+    <span>black</span>
```

Comment:
I think this might be more clear/useful if we put the whole syntax of the css variable here
We could perhaps even have the code example hidden by default then 
```suggestion
    <span>--sdds-black</span>
```